index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>PortfolioPage</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  </head>
+  <body>
+    
+    <main>
```

Comment:
- [ ] Please fix this weird behaviour when the user scrolls 

![image](https://user-images.githubusercontent.com/74744787/164725058-5cb06125-95bd-4d8b-97af-4a705e5af583.png)